gt/main.c

```diff
@@ -901,6 +1072,10 @@ cleanup_gt_instance(struct gt_instance *instance)
 
 	lua_close(instance->lua_state);
 	instance->lua_state = NULL;
+
+	/* If the pointer is NULL, the function does nothing. */
+	rte_ip_frag_table_destroy(instance->frag_tbl);
+	instance->frag_tbl = NULL;
```

Comment:
Preserve the reverse order, move the news lines to the beginning of the function.